### ISSUE TITLE
Zero Output Matrices in pre-trained models

### ISSUE THREAD

**Issue by chitwansaharia** [facebookresearch/fastText#976]
I tried loading bin files from https://fasttext.cc/docs/en/crawl-vectors.html for english, and use `fasttext dump <filename.bin> output` and `fasttext dump <filename.bin> input` for getting the output and input matrices respectively. However, the output matrix turns out to be a zero matrix. I also tried loading it in python, but it yields the same result. I tried german bin file as well, and faced the same issue. Fetching pretrained models (bin files) from https://fasttext.cc/docs/en/pretrained-vectors.html works fine. Is there something I am missing?

**Comment by Celebio** [comment]
Hi @chitwansaharia ,
The main purpose of these pre-trained models is to provide word vectors (the input matrix), rather than the classifier part of it (the output matrix). 

May I know what is your use-case?

Best regards,
Onur

**Comment by zhengshuo1** [comment]
> Hi @chitwansaharia ,
> The main purpose of these pre-trained models is to provide word vectors (the input matrix), rather than the classifier part of it (the output matrix).
> 
> May I know what is your use-case?
> 
> Best regards,
> Onur

Hello，May I ask you some questions?
The first one is, I want to evaluate the word-vector model trained by myself. Is it possible to set the backprop as false in the function of "real NegativeSamplingLoss::forward(
    const std::vector<int32_t>& targets,
    int32_t targetIndex,
    Model::State& state,
    real lr,
    bool backprop)"? and calculate the loss value in the function of Model::update?
The second one is,  is it possible to show the input and other parameters for training of the pre-trained word-vector model?
